test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Export/ExportDataTests.cs

```diff
@@ -188,6 +188,28 @@ public async Task GivenFhirServer_WhenDataIsExportedWithHistoryAndSoftDeletesNot
             await ExportAndSoftDeleteTestHelper(parallel: false, history: true, deletes: true);
         }
 
+        // No need to test both code paths for testing container is written to.
+        [Fact]
+        public async Task GivenFhirServer_WhenSummarizingHistoricalData_ThenCountIsReturnedWethHistory()
```

Comment:
Why is this in the export tests? This looks like it should be in HistoryTests.

Reply:
This was a suggestion above - I was concerned about E2E performance but can rewrite the test to go off less data. I haven't looked at history tests but will check it out.